tests/checkpoint/test_checkpoint.py

```diff
@@ -1213,10 +1213,6 @@ def test_newstyle_checkpoint_instantiates_and_produces_a_validation_result_when_
 
     assert len(context.validations_store.list_keys()) == 1
     assert results["success"] == True
-    try:
```

Comment:
Thank you for cleaning this pattern up @NathanFarmer.. btw do know what this try-except was trying to catch in our tests?

Reply:
There was no custom `__repr__` prior to PR #3680 and it was failing due to it not knowing how to represent DataFrames as strings.